ajax/libs/sketchpad/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "sketchpad",
+  "version": "0.1.0",
+  "description": "A simple sketchpad.",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/yiom/sketchpad.git"
+  },
+  "homepage": "https://yiom.github.io/sketchpad/",
+  "keywords": [
+    "sketchpad",
+    "javascript"
+  ],
+  "author": "Yiom <https://yiom.com.br>",
+  "license": "MIT",
+  "filename": "sketchpad.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/yiom/sketchpad.git",
```

Comment:
@sashberd Let's make the file structure as the followings:
```js
ajax/libs/sketchpad
├── 0.1.0
│   ├── scripts
│   │   ├── initialize.js
│   │   ├── initialize.min.js
│   │   ├── initialize.min.js.map
│   │   ├── sketchpad.js
│   │   ├── sketchpad.min.js
│   │   └── sketchpad.min.js.map
│   └── styles
│       └── sketchpad.css
└── package.json
```

The basePath which is stable should be `""`, `"dist"`, `"build"`, `"release"`...
We don't use `scripts`, `styles`, `js`, `css` which just under root as the basePath.

Thanks.